### ISSUE TITLE
[Light box media viewer]: QA: Keyboard focus is not navigating beyond 'Close' button in reverse navigation

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8994]
**Engineering info:**  
- Innovation Team engineer:
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2488
- DPO consulting engineer: N/A

-----

### Description

Page Element:
Close button

Issue Description:
Focus is not moving to the previous element after 'Close' button. It stays in the same element.

Expected:
On pressing Shift + tab key, focus should go to the previous element in the page.

User Impact:
When standard methods of keyboard navigation is not supported, this impact the user ability to interact with the page controls. This impacts Keyboard only users/Screen reader users

### Component(s) impacted

Light box media viewer

### Browser

Firefox

### Carbon for IBM.com version

v1.35.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
